.github/workflows/release_draft.yml

```diff
@@ -36,4 +36,47 @@ jobs:
           git config --global user.email "${{ github.actor }}@users.noreply.github.com"
 
       - name: checkout to latest minor release branch
-        run: git checkout release/${{ inputs.releaseVersion }}
+        run: |
+          git fetch origin release/${{ inputs.releaseVersion }}
+          git checkout release/${{ inputs.releaseVersion }}
+          git status
+
+      - name: bump patch version, commit and push
+        run: |
+          current_version=$(python -c 'from kili import __version__; print(__version__)')
+          new_version=$(./entrypoint.sh bump_version commit patch)
+          echo "NEW_PATCH_VERSION=$new_version" >> $GITHUB_ENV
+          echo "CURRENT_PATCH_VERSION=$current_version" >> $GITHUB_ENV
+          git push
+
+      - name: Tag patch commit and push
+        run: |
+          git tag -f -a ${{ env.NEW_PATCH_VERSION }} -m "Release ${{ env.NEW_PATCH_VERSION }}"
+          git push origin ${{ env.NEW_PATCH_VERSION }}
+
+      - name: Create draft release
+        run: gh release create ${{ env.NEW_PATCH_VERSION }} --draft --title "Release ${{ env.NEW_PATCH_VERSION }}" --generate-notes --notes-start-tag ${{ env.CURRENT_PATCH_VERSION }}
+        env:
+          GH_TOKEN: ${{ github.token }}
+
+      - name: Slack notification
+        id: slack
+        if: success()
+        uses: slackapi/slack-github-action@v1.18.0
+        with:
+          payload: |
+            {
+              "text": "Draft release ${{ env.NEW_PATCH_VERSION }} is ready: https://github.com/kili-technology/kili-python-sdk/releases",
```

Comment:
There should be a way to take the output of the "Create draft release" command: 
<img width="874" alt="image" src="https://user-images.githubusercontent.com/1255753/207048682-2a5f3a3a-506a-4bed-869f-c91f6aee2fdc.png">